types/v1/metrics.go

```diff
@@ -4,7 +4,35 @@ import "encoding/json"
 
 // VolumeMetrics is used to store the collected metrics
 // all the stats exposed by jiva stored into OpenEBSVolumeMetrics fields
-type VolumeMetrics struct {
+type VolumeMetrics []struct {
+	Name   string          `json:"name"`
```

Comment:
`types` package will be deprecated soon.
Can you try this instead:
`pkg/apis/openebs.io/v1alpha1/cas_volume_mretics.go`

Reply:
done